string  cmdType
float64 x_pos
float64 y_pos
float64 cur_theta
float64 x_dir
float64 y_dir
float64 des_theta
